src/Http/Controllers/Configuration/UserController.php

```diff
@@ -42,9 +46,74 @@ class UserController extends Controller
     public function getAll()
     {
 
-        $groups = $this->getAllGroups();
+        if(\request()->ajax()){
 
-        return view('web::configuration.users.list', compact('groups'));
+            $groups = $this->getAllFullUsers();
+
+            return DataTables::of($groups)
+                ->editColumn('refresh_token', function ($row) {
+                    return view('web::configuration.users.partials.refresh-token', compact('row'));
+                })
+                ->editColumn('name', function ($row) {
+
+                    if($row->id === 1)
```

Comment:
we rather should exclude the admin from the users list since it's a built-in user.

Reply:
done in service package PR